utils/Enum.hpp

```diff
@@ -3,4 +3,44 @@
 
 enum EHttpMethod { GET, POST, DELETE };
```

Comment:
아래와 동일하게 EHttpMethod도 HTTP_ 접두사를 붙이는 것이 좋을 것 같아요!

```suggestion
enum EHttpMethod { HTTP_GET, HTTP_POST, HTTP_DELETE };
```

Reply:
method enum도 변경하고 main문 예제에 반영했습니다!